make/framework.mk

```diff
@@ -57,8 +57,8 @@ FRAMEWORK_PUBLIC_HEADERS = $(FRAMEWORK_HEADERS)
 endif
 
 FRAMEWORK_DIR = $(DIST_FRAMEWORK_DIR)/$(FRAMEWORK_NAME).xcframework
-FRAMEWORK_HEADER = $(BUILD_DIR)/$(FRAMEWORK_NAME).h
-MODULE_MAP = $(BUILD_DIR)/module.modulemap
+FRAMEWORK_HEADER = $(BUILD_DIR)/$(FRAMEWORK_NAME)/$(FRAMEWORK_NAME).h
```

Comment:
It wasn't strictly necessary to move the `FRAMEWORK_HEADER` path from `$(BUILD_DIR)` to `$(BUILD_DIR)/$(FRAMEWORK_NAME)`. I made this change so that all build files for a given framework (i.e. both the framework header and the module map) are in the same place.